src/v/redpanda/admin_server.cc

```diff
@@ -936,6 +937,20 @@ void admin_server::register_config_routes() {
           return "";
       });
 
+    register_route_raw<superuser>(
+      ss::httpd::config_json::get_loggers, [](ss::const_req, ss::reply& reply) {
+          json::StringBuffer buf;
+          json::Writer<json::StringBuffer> writer(buf);
+          writer.StartArray();
```

Comment:
Can you make this an array of {"name": name} objects instead of an array of strings?  That way if we ever decide we want to put more in here (e.g. showing the current log level of each logger) we could do that.

Reply:
Done